### DISCUSSION TITLE
NASA's Operation IceBridge Data collection

### DISCUSSION THREAD

**Discussion by dr-vibhora** [qgreenland-workshop-2023-researcher/qgreenland-workshop-2023-researcher.github.io#142] [Dataset Ideas]
### Dataset URL

https://nsidc.org/data/icebridge/data

### Description

The IceBridge data collection is a continuation of ICESat/GLAS satellite altimetry mission (ended in 2010) for continuous observations of Earth’s Polar realm between 2009-2020. The mission involved >1000 airborne surveys (highest in the polar regions) to map coastal Greenland, coastal Antarctica, the Antarctic Peninsula, interior Antarctica, the southeast Alaskan glaciers, and Antarctic and Arctic sea ice. The mission utilized a fleet of aircraft fitted with a suite of instruments including radar sounders, gravimeters, magnetometers and mapping cameras. Data products includes glacier and ice sheet elevation, glacier and ice sheet topography, sea ice elevation, sea ice thickness, terrain elevation, bedrock topography, snow/ice temperature, radar imagery, visible imagery, meteorological observations and calibration data.

REASON why I am so much interested in the data is because of the disparate dataets available through this mission which could imporve my understanding of geospatial analysis and contribute to QGreenland project

**Comment by lauralindzey** [comment]
I'm also interested in this! I've put a geopackage file in group four's directory containing all of the CReSIS flightlines in Greenland, which covers most (all?) of the Greenland OIB airborne data.  (UTIG did some OIB flights in Antarctica; I'm not sure if they had any in Greenland.)
